### ISSUE TITLE
PushNotification.localNotification / localNotificationSchedule not working on rn0.60 (Android)

### ISSUE THREAD

**Issue by adrianchancy** [zo0r/react-native-push-notification#1156]
I have linked the project and configured all mentioned in
(https://github.com/zo0r/react-native-push-notification#android-manual-installation)

But the notification is not shown with below example:
(https://github.com/zo0r/react-native-push-notification#scheduled-notifications)

React Native Version: 0.60.4
Android: 8.0 (Both in emulator and real device)
iOS works well.

Does anyone face the same issue?

**Comment by swaroopa94** [comment]
@adrianchancy Hi did you fix the issue I too have the same issue in android

**Comment by adrianchancy** [comment]
> @adrianchancy Hi did you fix the issue I too have the same issue in android

Not yet
Previously, I could make it works in Android with earlier versions of this library and react native itself. But... I forget the exact versions.

**Comment by swaroopa94** [comment]
@adrianchancy okey then how u r managing now then should I install the previous versions then?

**Comment by adrianchancy** [comment]
@swaroopa94 
It is still pending, as it is just for a personal project.
Anyway, I will try again in these few days to see if there is any way out. Maybe... downgrading is an option, but I am not sure.

**Comment by swaroopa94** [comment]
@adrianchancy but mine is not sample so thats why I am asking is there any alternative
I will also try by downgrading the version

**Comment by adrianchancy** [comment]
@swaroopa94 
I found I am missing something in AndroidManifest.xml

It is mentioned in https://github.com/zo0r/react-native-push-notification#android-manual-installation. I ignored it because I thought for local push, we don't need to define channel, but I was wrong.

`<application ....>
        <meta-data  android:name="com.dieam.reactnativepushnotification.notification_channel_name"
                android:value="YOUR NOTIFICATION CHANNEL NAME"/>
        <meta-data  android:name="com.dieam.reactnativepushnotification.notification_channel_description"
                    android:value="YOUR NOTIFICATION CHANNEL DESCRIPTION"/>`

For channel name and channel description, I entered "localpush". Run "react-native run-android". Then, it works.

**Comment by swaroopa94** [comment]
@adrianchancy we can give any channel name to it?

**Comment by spacewaffle** [comment]
@adrianchancy thank you! works for me. @swaroopa94 you can give the channel any name or description you want. Users will see it in their settings app when they go to change notification settings.

**Comment by swaroopa94** [comment]
@spacewaffle and @adrianchancy  thanks Its working fine now

**Comment by ahmadallo1** [comment]
@spacewaffle  @spacewaffle same issue localNotificationSchedule not working(localNotification is working properly) on android 0.60.5 & react-native-push-notification 3.1.1
also I have updated the AndroidManifest.xml, any idea how to fix this

**Comment by d0rf47** [comment]
I FOUND A SOLUTION!!!!!

So in your android/app/src/main/assets/index.android.bundle file you will find the compressed bundled src code which gets loaded into the main Java class. You will need to ctrl f for your notifications and locate the place where you set the date for the scheduling of the notification.  I noticed that when I made changes the my notificationManager.js file, the changes WERE NOT reflected in  bundle file.  So  All i did was change the code in the bundle to reflect the code in my .js file and build and deploy the app and it was working.

1 Caveat though, I am not sure why this is happening know & i think it has something to do with TimeZone offsets, but after I made the change for the deployment, it works in deploymen but when I debug other parts of the application in debug mode the notifications don't work. But if I make changes and rebuild, as long as I don't mess with the notifications It still works when I rebuild & deploy.
